### PR TITLE
WD-3110 build `/18-04/gcp`

### PR DESCRIPTION
**Pull request by pedoch**
## Done

- Build the `/18-04/gcp` page

## QA

- Check out the [demo here](https://ubuntu-com-12802.demos.haus/18-04/gcp) and compare it with the [copy doc](https://docs.google.com/document/d/1d62l-BdsgXjs0PZ_JuxJ0u_xG7lUNqyFVBN-lCvSt00/edit#) and with the [Azure page](https://ubuntu-com-12802.demos.haus/18-04/azure) for design review.

## Issue / Card

- [WD-3110](https://warthogs.atlassian.net/browse/WD-3110)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-3110]: https://warthogs.atlassian.net/browse/WD-3110?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by juanruitina** [comment]
Some comments:

- Under "What are your options?" there are two CTA buttons in the copy doc that show up in the demo as links. Can you please change this? (I've asked Hugo to confirm that he is fine with the second one being a default/white button).
- This link "[more information on Upgrading to Ubuntu Pro below](https://docs.google.com/document/d/1d62l-BdsgXjs0PZ_JuxJ0u_xG7lUNqyFVBN-lCvSt00/edit#bookmark=id.de99h0no3f6y)." heads to the Google doc instead of to the section in the page, can you tweak?
- I see we refer to the bubble in the meganav (under the Ubuntu logo) as "18-04" instead of "18.04". Can we fix this?

File: templates/18-04/gcp.html [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-muted-heading">Application packages</h3>
+        </div>
+        <div class="col-6 col-medium-4">
+          <p>
+            Canonical provides application extended security maintenance to the binary packages that reside in Main and Universe. A few commonly-used packages include:
+          </p>
+          <ul class="p-list--divided">
+            <li class="p-list__item has-bullet">MySQL - 5.7</li>
```
Comment (mtruj013) [inline templates/18-04/gcp.html:232]:
```suggestion
            <li class="p-list__item has-bullet">MySQL &ndash; 5.7</li>
```

File: templates/18-04/gcp.html [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+            <li class="p-list__item has-bullet">Python - 2.7</li>
```
Comment (mtruj013) [inline templates/18-04/gcp.html:233]:
```suggestion
            <li class="p-list__item has-bullet">Python &ndash; 2.7</li>
```

File: templates/18-04/gcp.html [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+            <li class="p-list__item has-bullet">PostgreSQL - 10</li>
```
Comment (mtruj013) [inline templates/18-04/gcp.html:234]:
```suggestion
            <li class="p-list__item has-bullet">PostgreSQL &ndash; 10</li>
```

File: templates/18-04/gcp.html [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+            <li class="p-list__item has-bullet">Ruby - 2.5</li>
+            <li class="p-list__item has-bullet">Nodejs - 8.10</li>
```
Comment (mtruj013) [inline templates/18-04/gcp.html:236]:
```suggestion
            <li class="p-list__item has-bullet">Nodejs &ndash; 8.10</li>
```

File: templates/18-04/gcp.html [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+            <li class="p-list__item has-bullet">PHP - 7.2</li>
```
Comment (mtruj013) [inline templates/18-04/gcp.html:237]:
```suggestion
            <li class="p-list__item has-bullet">PHP &ndash; 7.2</li>
```

File: templates/18-04/gcp.html [hunk 6]
```diff
@@ -0,0 +1,312 @@
...
+            <li class="p-list__item has-bullet">ROS - Melodic</li>
```
Comment (mtruj013) [inline templates/18-04/gcp.html:238]:
```suggestion
            <li class="p-list__item has-bullet">ROS &ndash; Melodic</li>
```

**Review by mtruj013 — CHANGES_REQUESTED** [review]
Thanks @pedoch ! A few comments in the code:

File: templates/18-04/gcp.html [hunk 7]
```diff
@@ -0,0 +1,312 @@
...
+        <span class="p-pull-quote__citation">Andy Parker, Engineering Manager, TIM</span>
+        <a href="/engage/case-study-acuris">Read the case study&nbsp;&rsaquo;</a>
+      </blockquote>
+    </div>
+  </div>
+</section>
+
+<hr class="is-fixed-width" />
+
+<section class="p-strip is-deep">
```
[the remark this answers is not in the document]
Comment (pedoch) [reply templates/18-04/gcp.html:304]: Uh no, the new strip doesn't follow the new design. This is what is on `/18-04` (which Lyubo said I should use) as a guide.

**Comment by pedoch** [comment]
@mtruj013 could you take another look?

**Comment by juanruitina** [comment]
Minor tweaks to those two buttons:

* Please swap them, "In place upgrade" should come first (colour is correct)
* Remove "for" in  "[Get started with Ubuntu Pro for 18.04 LTS](https://ubuntu-com-12802.demos.haus/gcp/pro)" to avoid line break in mobile (it also seems more consistent wording)
* Spacing between them is a bit weird in relation with the text above. Maybe just place both buttons on the same paragraph, it seems to do the trick and they will still be stacked.

**Review by mtruj013 — APPROVED** [review]
LGTM

**Comment by juanruitina** [comment]
UX +1. (I base this mostly on consistency with the other 18.04 pages, that went live without a UX review)

**Pull request merged**